Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/internal/Hole_filling/Triangulate_hole_polyline.h

```diff
@@ -1318,88 +1360,81 @@ triangulate_hole_polyline_with_cdt(const PointRange1& points,
      return false;
 
   Lookup_table_map<int> lambda(static_cast<int>(size), -1);
+
   // Create and fill the cdt_2.
   typedef CGAL::Triangulation_vertex_base_with_info_2<std::size_t, P_traits> Vb;
   typedef CGAL::Triangulation_face_base_with_info_2<bool, P_traits>          Fbi;
   typedef CGAL::Constrained_triangulation_face_base_2<P_traits, Fbi>         Fb;
-  typedef CGAL::Triangulation_data_structure_2<Vb,Fb>                        TDS;
-  typedef CGAL::No_constraint_intersection_tag                               Itag; // If the polygon is simple, there should be no intersection.
+  typedef CGAL::Triangulation_data_structure_2<Vb, Fb>                       TDS;
+  // If the polygon is simple, there should be no intersection.
+  typedef CGAL::No_constraint_intersection_tag                               Itag;
   typedef CGAL::Constrained_Delaunay_triangulation_2<P_traits, TDS, Itag>    CDT;
   P_traits cdt_traits(avg_normal);
   CDT cdt(cdt_traits);
 
-  std::vector< std::pair<Point_3,std::size_t> > points_and_ids;
+  std::vector< std::pair<Point_3, std::size_t> > points_and_ids;
   points_and_ids.reserve(size);
-  std::vector<typename CDT::Vertex_handle> vertices(size);
   for (std::size_t i = 0; i < size; ++i)
-  {
-    points_and_ids.push_back(std::make_pair(P[i],i));
-  }
+    points_and_ids.push_back(std::make_pair(P[i], i));
+
+  std::vector<typename CDT::Vertex_handle> vertices(size);
   cdt.insert(points_and_ids.begin(), points_and_ids.end());
   for (typename CDT::Vertex_handle v : cdt.finite_vertex_handles())
```

Comment:
I find it a bad idea not to put parenthesis, and here you even replace them with an empty line.

Reply:
I can add them back. I just shortened the code. The parenthesis should not affect anything if I am not mistaken. And the empty line makes the code more readable IMHO.